src/sysusers/sysusers.c

```diff
@@ -1041,15 +1041,19 @@ static int uid_is_ok(
                 r = getpwuid_malloc(uid, /* ret= */ NULL);
                 if (r >= 0)
                         return 0;
-                if (r != -ESRCH)
+                if (ERRNO_IS_NEG_DISCONNECT(r))
```

Comment:
i wouldn't even special case that. Just do this in all error cases that arent's ESRCH, whether connection errors or not

Reply:
ok, done